src/app/pages/LendingPage/components/CurrencyContainer/UserLendingInfo.tsx

```diff
@@ -35,7 +35,10 @@ export const UserLendingInfo: React.FC<IUserLendingInfoProps> = ({
   const { t } = useTranslation();
   const account = useAccount();
   const asset = lendingPool.getAsset();
-  const { value: rewards } = useLiquidityMining_getUserAccumulatedReward(
+  const {
+    value: rewards,
+    loading: rLoading,
```

Comment:
Please rename to `rewardsLoading`, I don't like those abbreviations in the code.